### PR TITLE
Update defaults.ts with HydraDX re-genesis hash

### PR DESCRIPTION
**Pull request by jak-pan**

### CODE REVIEW

File: packages/networks/src/defaults.ts [hunk 1]
```diff
@@ -14,6 +14,9 @@ export const knownGenesis: KnownGenesis = {
   edgeware: [
     '0x742a2ca70c2fda6cee4f8df98d64c4c670a052d9568058982dad9d5a7a135c5b'
   ],
+  hydradx: [
```
Comment (jacogr) [inline packages/networks/src/defaults.ts:17]: Do you have the precious hash as well - accounts tied to the previous hash can also match to the new if both supplied.

Comment (jacogr) [reply packages/networks/src/defaults.ts:17]: If this is the current, all ok.

File: packages/networks/src/defaults.ts [hunk 2]
```diff
@@ -14,6 +14,9 @@ export const knownGenesis: KnownGenesis = {
...
+    '0x3d75507dd46301767e601265791da1d9cb47b6ebc94e87347b635e5bf58bd047'
```
Comment (jacogr) [inline packages/networks/src/defaults.ts:18]:
```suggestion
    '0x3d75507dd46301767e601265791da1d9cb47b6ebc94e87347b635e5bf58bd047',
    '0x0ed32bfcab4a83517fac88f2aa7cbc2f88d3ab93be9a12b6188a036bf8a943c2'
```

Adding the current genesisHash from RPC.

Comment (jak-pan) [reply packages/networks/src/defaults.ts]: Yes that's the previous one. Thanks!

**Review by jacogr — APPROVED** [review]

**Comment by jacogr** [comment]
I will roll this out to the apps UI in a couple of hours - it may actually close the loop and just allow everything "to work", old or news.

- extension sets the hash, can be old or new
- apps UI will match hashes (either old or new)

So when the user uses the old or updates, he/she should already be able to set and use it in the extension after it.

(It should have been in already, but running a bit late, so can promise a couple of hours, will link to this issue)

**Comment by jak-pan** [comment]
> I will roll this out to the apps UI in a couple of hours - it may actually close the loop and just allow everything "to work", old or news.
> 
> * extension sets the hash, can be old or new
> * apps UI will match hashes (either old or new)
> 
> So when the user uses the old or updated new, he/she should already be able to set and use it in the extension after it.
> 
> (It should have been in already, but running a bit late, so can promise a couple of hours, will link to this PR)

Thank you very much! You're quick 👀

**Pull request merged**